build.sbt

```diff
@@ -0,0 +1,105 @@
+import sbt._
+
+val scala2 = "2.13.8"
+
+// val scala3 = "3.1.0"
+
+ThisBuild / organization := "com.caesars"
+ThisBuild / scalaVersion := scala2
+// ThisBuild / crossScalaVersions := List(scala2, scala3)
+ThisBuild / versionScheme := Some("early-semver")
+// usually, this is set by sbt-dynver. For some cases, like an out-of-bound docker image, allow override
+ThisBuild / version ~= (v => sys.env.getOrElse("SBT_VERSION_OVERRIDE", v))
+
+addCommandAlias("lint", "; scalafmtSbt; scalafmtAll")
+
+lazy val root = (project in file("."))
+  .settings(
+    name := "trace4cats-zio-extras",
+  )
+  .aggregate(core, sttp, zhttp, testkit)
+
+lazy val core = mkModule("core")
+  .settings(
+    libraryDependencies ++= List(
+      // TODO: replace Calvin's version with the canonical one once its released
+      "com.github.kaizen-solutions.trace4cats-newrelic" %% "trace4cats-newrelic-http-exporter" % "0.12.2",
```

Comment:
Per the TODO, this `com.github.kaizen-solutions.trace4cats-newrelic` is a copy of a to-be released `trace4cats-newrelic` module?

Reply:
that's my understanding in speaking with Calvin. this is a self-published stopgap until the upstream component is released.